s3tests_boto3/functional/test_s3.py

```diff
@@ -2817,6 +2817,53 @@ def test_post_object_upload_size_below_minimum():
 
     r = requests.post(url, files=payload, verify=get_config_ssl_verify())
     assert r.status_code == 400
+    assert r.error_code == 'EntityTooSmall'
+
+def test_post_object_upload_size_rgw_chunk_size_bug():
+    # TODO: if this value is different in Teuthology runs, this would need tuning
+    _rgw_max_chunk_size = 4 * 2**20 # 4MiB
+    min_size = _rgw_max_chunk_size * 2
+    max_size = _rgw_max_chunk_size * 3
+    # [(chunk),(small)]
+    test_payload_size = min_size * 2 + 200 # extra bit to push it over the chunk boundary
+    # it should be valid when we run this test!
+    assert test_payload_size > min_size
+    assert test_payload_size < max_size
```

Comment:
double-checking my math here..
```
rgw_max_chunk_size = 4M
min_size = 8M
max_size = 12M
test_payload_size = 16M + 200
```
so won't this `max_size` assertion fail?

in the interest of a minimal reproducer, could we just use `test_payload_size = _rgw_max_chunk_size + 200`?

Reply:
oops, editing fail when I cleaned this from internal version

Reply:
Fixed